core/dvector.h

```diff
@@ -232,17 +232,19 @@ class DVector {
 	}
 
 
-	void remove(int p_index) {
+	T remove(int p_index) {
 
+		T ret = get(p_index);
```

Comment:
Yah, no, this isn't the problem, I can't pinpoint it. If in godot we use var arr = [2, 3, 4]; arr[10] it lists the error in the debugger, but with my modification, if I try arr.remove(10) it just closes the game and gives the error in the terminal as explained above and I don't know how to fix it. Hopefully someone does :)